packages/valory/skills/market_manager_abci/bets.py

```diff
@@ -136,18 +138,59 @@ class Bet:
     prediction_response: PredictionResponse = dataclasses.field(
         default_factory=get_default_prediction_response
     )
-    invested_amount: float = 0.0
     position_liquidity: int = 0
     potential_net_profit: int = 0
     processed_timestamp: int = 0
-    n_bets: int = 0
     queue_status: QueueStatus = QueueStatus.FRESH
+    # a mapping from vote to investment amounts
+    investments: Dict[str, List[int]] = dataclasses.field(default_factory=dict)
```

Comment:
Invesments will be treated as a dataclass when decoding is being done, a hook will have to be setup for this or will have to make investments as a data class